### ISSUE TITLE
iOS App Launch Issue: iPhone 8 

### ISSUE THREAD

**Issue by mikaelajoyce** [engagementlab/co-creation-of-vr-360#1]
When the app button is clicked on an iPhone, it automatically opens into the VR/AR camera. It skips past a menu, call to action buttons, etc. There is no way to click out of the camera other than closing the app.

**Comment by johnnycrich** [comment]
Are you seeing this screen at all @mikaelajoyce?
[Screen Shot 2021-02-11 at 4.50.20 PM.png](https://images.zenhubusercontent.com/57b5c528e40e5714b16d8429/6e8bae22-6632-4567-9529-c36b0428101f)

As far as no way to leave the camera other than closing the app, agreed this is a UX issue.

**Comment by mikaelajoyce** [comment]
Yes - once I click "start" it goes directly into the camera mode. The camera mode can't be navigated out of without closing the app. No buttons/menu functions appear.

**Comment by johnnycrich** [comment]
Yes, that is how the app works as of me receiving the code --- it did not even have a title screen before I added one.

**Comment by johnnycrich** [comment]
@mikaelajoyce I am adding a back button to a new build, coming shortly.

**Comment by johnnycrich** [comment]
@mikaelajoyce Please let me know if the new release (0.9 (3)) resolves.
